web_sdk/BUILD.gn

```diff
@@ -100,21 +101,76 @@ copy("web_ui_library") {
   outputs = [ "$root_out_dir/flutter_web_sdk/{{source_file_part}}" ]
 }
 
-# Compile the unsound DDC SDK's summary.
-prebuilt_dart_action("flutter_dartdevc_kernel_sdk_outline") {
-  deps = [
-    "//third_party/dart:create_sdk",
-    "//third_party/dart/pkg:pkg_files_stamp",
-    "//third_party/dart/utils/dartdevc:dartdevc_files_stamp",
-    "//third_party/dart/utils/dartdevc:dartdevc_sdk_patch_stamp",
-  ]
+template("_dartdevc") {
```

Comment:
I don't know what these templates are supposed to do and neither are their arguments documented. It's possible to infer both after reading the implementation but a short docstring might be nice.

Reply:
Added comments.